src/types.db

```diff
@@ -175,6 +176,7 @@ ping_droprate           value:GAUGE:0:100
 ping_stddev             value:GAUGE:0:65535
 players                 value:GAUGE:0:1000000
 power                   value:GAUGE:0:U
+power_battery           value:GAUGE:U:U
```

Comment:
Actually, scratch that, I'm going to make this change in a bugfix branch and merge it to master from there, so older versions get the fix, too.